### PR TITLE
LPS-199057

### PR DESCRIPTION
**Pull request by CaiocSiqueira**
https://liferay.atlassian.net/browse/LPS-199057

PASSED Testing on: https://github.com/CaiocSiqueira/liferay-portal/pull/69

### CODE REVIEW

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by CaiocSiqueira** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/search/SearchTuning.path [hunk 1]
```diff
@@ -33,6 +33,11 @@
 	<td>//div[@data-testid='add-result-modal']//input[contains(@aria-label,'Search')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>RESULT_RANKINGS_ADD_RESULT_TOP_RESULT</td>
+	<td>//li[contains(@class,'list-group-item')]</td>
+	<td></td>
+</tr>
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/search/SearchTuning.path:40]: This path actually matches all results in the "Add Results" modal.

Try creating a path like `SearchPage#RESULT_TITLE_SPECIFIC_ORDER`. This path takes the specific order of the result so it's reusable in other scenarios.

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by xbrianlee** [comment]
See additional change: https://github.com/xbrianlee/liferay-portal/pull/1364/commits/fb284c82c6e6b38768b8db991566387fcd10bc49

**Pull request closed without merging**